doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -321,6 +321,12 @@ Event manager
 
 * Added:
 
+  * Universal hooks for event manager initialization, event submission, preprocessing and postprocessing.
+    This includes implementation of hooks registering macros:
+    - :c:macro:`EVENT_MANAGER_HOOK_POSTINIT_REGISTER`
+    - :c:macro:`EVENT_HOOK_ON_SUBMIT_REGISTER`, :c:macro:`EVENT_HOOK_ON_SUBMIT_REGISTER_FIRST`, :c:macro:`EVENT_HOOK_ON_SUBMIT_REGISTER_LAST`
+    - :c:macro:`EVENT_HOOK_PREPROCESS_REGISTER`, :c:macro:`EVENT_HOOK_PREPROCESS_REGISTER_FIRST`, :c:macro:`EVENT_HOOK_PREPROCESS_REGISTER_LAST`
+    - :c:macro:`EVENT_HOOK_POSTPROCESSS_REGISTER`, :c:macro:`EVENT_HOOK_POSTPROCESSS_REGISTER_FIRST`, :c:macro:`EVENT_HOOK_POSTPROCESSS_REGISTER_LAST`
```

Comment:
@rakons , is this a typo or intended triple S? `EVENT_HOOK_POSTPROCE[SSS]_REGISTER_LAST`

Reply:
Three S are not intentional here. I will fix it.